kube-hunter.py

```diff
@@ -74,23 +76,32 @@ def parse_docs(hunter, docs):
     for i, line in enumerate(docs):
         docs[i] = line.strip()
     return docs[0], ' '.join(docs[1:]) if len(docs[1:]) else "<no documentation>"
-    
+
+
 def list_hunters():
     print("\nPassive Hunters:\n----------------")
     for i, (hunter, docs) in enumerate(handler.passive_hunters.items()):
         name, docs = parse_docs(hunter, docs)
-        print("* {}\n  {}\n".format( name, docs))
+        print("* {}\n  {}\n".format(name, docs))
 
     if config.active:
         print("\n\nActive Hunters:\n---------------")
         for i, (hunter, docs) in enumerate(handler.active_hunters.items()):
             name, docs = parse_docs(hunter, docs)
             print("* {}\n  {}\n".format( name, docs))
-        
 
+
+global hunt_started_lock
+hunt_started_lock = threading.Lock()
+hunt_started_lock.acquire()
 hunt_started = False
+hunt_started_lock.release()
+
```

Comment:
I think this code will only get called once (when the module is loaded) so you don't really need to grab the lock around it